### ISSUE TITLE
Test object should be in test method

### ISSUE THREAD

**Issue by ghost** [divinenickname/utgen-kotlin-idea-plugin#2]
Hello! I have small issue in your implementation of test suites.

Generated code:
```
package org.pkg

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class MyAwesomeTest {
  private val obj: MyAwesome = MyAwesome()

  @Test
  public fun yoyo_goldencase() {
    TODO("Implement")
    val expected = Int()
    val actual = obj.yoyo()

    Assertions.assertEquals(expected, actual)
  }
}
```

`  private val obj: MyAwesome = MyAwesome()` - this instantiation should be in each tests method because some methods may mutate object's state. Junit execute tests in random order by default.

Version 1.1.0

**Comment by divinenickname** [comment]
Should be fixed in 1.2.0 - 1.2.1 (not released in marketplace yet)

**Comment by divinenickname** [comment]
fixed in 1.2.2
